cg/models/balsamic/config.py

```diff
@@ -62,34 +59,29 @@ def validate_genome_version(cls, value: Path, values: dict) -> str:
 
         return str(values["reference_genome"]).split("/")[-3]
 
-    @validator("reference_access_date")
-    def validate_reference_date(cls, value: Path) -> datetime:
-        """Formats the reference date"""
-
-        return datetime.strptime(os.path.basename(value), "%Y-%m-%d %H:%M:%S")
-
 
 class BalsamicConfigPanel(BaseModel):
-    """BALSAMIC attributes of a PANEL BED file if it's provided
+    """Balsamic attributes of a panel BED file
 
     Attributes:
-        capture_kit: string representation of a path to the PANEL BED file
+        capture_kit: string representation of a panel BED filename
         capture_kit_version: capture kit version
-        chrom: list of chromosomes in PANEL BED
+        chrom: list of chromosomes in the panel BED file
     """
 
-    capture_kit: Path
+    capture_kit: Union[str, Path]
```

Comment:
That's a good point. The question here is how to type hint a pydantic variable that changes its type during validation?

Here it's not a problem because it's actually a `str` what is passed, but there are some other models in the delivery report API that are using Union for this purpose 🤔